tests/cli-configuration.spec.js

```diff
@@ -15,10 +15,12 @@
  */
 
 'use strict';
-
 const _ = require('lodash');
 const m = require('mochainon');
 const configuration = require('../lib/cli/configuration');
+const packageJSON = require('../package.json');
+const path = require('path');
+const internalConfPath = path.join(__dirname, `../${packageJSON.name}.conf.js`);
```

Comment:
Is it worth inserting a blank line above to visually separate this from the `require` lines?